### PR TITLE
DS-698 New brand logos

### PR DESCRIPTION
**Pull request by jeremy-davis-sonarsource**
Designs [are here](https://www.figma.com/design/3MMKlRsuGgH5fhU95zrVEv/branch/A1fMWy7ai4FytHunPROcVw/(0.9.0-Echoes)-Foundations?node-id=2856-6578&node-type=symbol&m=dev)

### CODE REVIEW

File: src/components/logos/LogoSonar.tsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  const intl = useIntl();
+
+  return hasText ? (
+    <svg
+      fill="none"
+      height="48"
+      ref={ref}
+      role="img"
+      viewBox="0 0 152 48"
+      width="152"
```
Comment (gregaubert) [inline src/components/logos/LogoSonar.tsx:37]: Could having the height and width hardcoded like this prevent us to use the logos to just fill some available space? or just passing `height="100%" width="100%"` would work fine?

**Review by gregaubert — APPROVED** [review]
Looks good!

By any chance did you pass the SVG through some size optimizer like [svgomg](https://svgomg.net/)?

Also, I don't think you can do much, it might be more on the side of @marciopmoreira6. It might be just my screen/computer but it seems like there is a lot of aliasing on the new SQ logos, especially on the colored parts:

image

And even zooming in, the lines don't seem very clean.

File: src/components/logos/LogoSonar.tsx (hunk 1 above)
Comment (jeremy-davis-sonarsource) [reply src/components/logos/LogoSonar.tsx:37]: Hmm, I didn't think we'd have variable sizes on these. I'll check with Marcio

**Comment by david-cho-lerat-sonarsource** [comment]
> Looks good!
> 
> By any chance did you pass the SVG through some size optimizer like [svgomg](https://svgomg.net/)?
> 
> Also, I don't think you can do much, it might be more on the side of @marciopmoreira6. It might be just my screen/computer but it seems like there is a lot of aliasing on the new SQ logos, especially on the colored parts:
> 
> image
> 
> And even zooming in, the lines don't seem very clean.

`yarn dev` didn't start for me because of the `<Svg></svg>` mismatch in `LogoSonar`, but once fixed I have clear-cut images, no aliasing
image

**Review by gregaubert — APPROVED** [review]
Code looks good!

I still have a bit of aliasing/bleeding, but that might just be my computer/screen...

**Comment by david-cho-lerat-sonarsource** [comment]
> I still have a bit of aliasing/bleeding, but that might just be my computer/screen...
Yes, I believe it's on your side, it looks fine for me

**Review by david-cho-lerat-sonarsource — APPROVED** [review]
LGTM :+1:

File: src/components/logos/LogoSonar.tsx [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+import { forwardRef } from 'react';
```
Comment (david-cho-lerat-sonarsource) [inline src/components/logos/LogoSonar.tsx:20]: NP: we should update the ESLint rule to have consistent newlines under the headers, like in SQS :stuck_out_tongue_closed_eyes:

**Pull request merged**